### PR TITLE
[Enhancement] Support show sync stream load

### PR DESCRIPTION
**Pull request by smartlxh**
## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #

## Problem Summary(Required):

When the session variable `enable_load_profile ` is set to true #21441, the stream load will be recorded. Then we can see stream load through `show stream load` and distinguish them by `type` including `STREAM_LOAD`, `ROUTINE_LOAD`, and `PARALLEL_STREAM_LOAD`.
Type `STREAM_LOAD` indicates the load is created by `/api/dateabase/table/_stream_load`, `ROUTINE_LOAD` indicates  the load corresponds to a routine load task as well as `PARALLEL_STREAM_LOAD` indicates the load is created with the way of 
pr #12182

image

Support filter stream load by type through condition

image

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 22 / 94 (23.40%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadJob.java|0|6|00.00%|[792, 991, 992, 993, 994, 995]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|40|00.00%|[1164, 1165, 1166, 1169, 1267, 1268, 1282, 1283, 1284, 1285, 1286, 1287, 1300, 1301, 1302, 1303, 1304, 1305, 1427, 1428, 1429, 1431, 1433, 1434, 1436, 1438, 1439, 1440, 1441, 1442, 1443, 1449, 1450, 1452, 1454, 1455, 1456, 1457, 1458, 1459]|
|:large_blue_circle:|com/starrocks/load/loadv2/ManualLoadTxnCommitAttachment.java|0|2|00.00%|[47, 70]|
|:large_blue_circle:|com/starrocks/load/routineload/KafkaTaskInfo.java|0|1|00.00%|[200]|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadTaskInfo.java|0|4|00.00%|[151, 235, 236, 238]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadManager.java|2|5|40.00%|[375, 376, 475]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadTask.java|12|27|44.44%|[1012, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1095, 1284, 1285, 1292, 1306, 1308, 1388]|
|:large_blue_circle:|com/starrocks/load/routineload/RLTaskTxnCommitAttachment.java|1|2|50.00%|[121]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowStreamLoadStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadFunctionalExprProvider.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 22 / 93 (23.66%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadJob.java|0|5|00.00%|[991, 992, 993, 994, 995]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|40|00.00%|[1165, 1166, 1167, 1170, 1268, 1269, 1283, 1284, 1285, 1286, 1287, 1288, 1301, 1302, 1303, 1304, 1305, 1306, 1428, 1429, 1430, 1432, 1434, 1435, 1437, 1439, 1440, 1441, 1442, 1443, 1444, 1450, 1451, 1453, 1455, 1456, 1457, 1458, 1459, 1460]|
|:large_blue_circle:|com/starrocks/load/loadv2/ManualLoadTxnCommitAttachment.java|0|2|00.00%|[51, 74]|
|:large_blue_circle:|com/starrocks/load/routineload/KafkaTaskInfo.java|0|1|00.00%|[200]|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadTaskInfo.java|0|4|00.00%|[151, 235, 236, 238]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadManager.java|2|5|40.00%|[375, 376, 475]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadTask.java|12|27|44.44%|[1012, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1095, 1284, 1285, 1292, 1306, 1308, 1388]|
|:large_blue_circle:|com/starrocks/load/routineload/RLTaskTxnCommitAttachment.java|1|2|50.00%|[128]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowStreamLoadStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadFunctionalExprProvider.java|6|6|100.00%|[]|

**Review by chaoyli — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 22 / 94 (23.40%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadJob.java|0|5|00.00%|[991, 992, 993, 994, 995]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|41|00.00%|[1165, 1166, 1167, 1168, 1171, 1269, 1270, 1284, 1285, 1286, 1287, 1288, 1289, 1302, 1303, 1304, 1305, 1306, 1307, 1429, 1430, 1431, 1433, 1435, 1436, 1438, 1440, 1441, 1442, 1443, 1444, 1445, 1451, 1452, 1454, 1456, 1457, 1458, 1459, 1460, 1461]|
|:large_blue_circle:|com/starrocks/load/loadv2/ManualLoadTxnCommitAttachment.java|0|2|00.00%|[51, 74]|
|:large_blue_circle:|com/starrocks/load/routineload/KafkaTaskInfo.java|0|1|00.00%|[200]|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadTaskInfo.java|0|4|00.00%|[151, 235, 236, 238]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadManager.java|2|5|40.00%|[375, 376, 475]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadTask.java|12|27|44.44%|[1012, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1095, 1284, 1285, 1292, 1306, 1308, 1388]|
|:large_blue_circle:|com/starrocks/load/routineload/RLTaskTxnCommitAttachment.java|1|2|50.00%|[128]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowStreamLoadStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadFunctionalExprProvider.java|6|6|100.00%|[]|

**Review by caneGuy — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java [hunk 1]
```diff
@@ -1261,15 +1280,35 @@ public void setTxnId(long txnId) {
...
+    public String getStringByType() {
+        switch (this.type) {
+            case ROUTINE_LOAD:
+                return "ROUTINE_LOAD";
+            case STREAM_LOAD:
+                return "STREAM_LOAD";
+            case PARALLEL_STREAM_LOAD:
+                return "PARALLEL_STREAM_LOAD";
+            default:
+                return "UNKOWN";
```
Comment (mxdzs0612) [inline fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java:1308]: UNKOWN？

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 22 / 94 (23.40%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadJob.java|0|5|00.00%|[991, 992, 993, 994, 995]|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|41|00.00%|[1090, 1091, 1092, 1093, 1096, 1194, 1195, 1209, 1210, 1211, 1212, 1213, 1214, 1227, 1228, 1229, 1230, 1231, 1232, 1354, 1355, 1356, 1358, 1360, 1361, 1363, 1365, 1366, 1367, 1368, 1369, 1370, 1376, 1377, 1379, 1381, 1382, 1383, 1384, 1385, 1386]|
|:large_blue_circle:|com/starrocks/load/loadv2/ManualLoadTxnCommitAttachment.java|0|2|00.00%|[51, 74]|
|:large_blue_circle:|com/starrocks/load/routineload/KafkaTaskInfo.java|0|1|00.00%|[200]|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadTaskInfo.java|0|4|00.00%|[151, 235, 236, 238]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadManager.java|2|5|40.00%|[375, 376, 475]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadTask.java|12|27|44.44%|[1012, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1095, 1284, 1285, 1292, 1306, 1308, 1388]|
|:large_blue_circle:|com/starrocks/load/routineload/RLTaskTxnCommitAttachment.java|1|2|50.00%|[128]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowStreamLoadStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/load/streamload/StreamLoadFunctionalExprProvider.java|6|6|100.00%|[]|

**Review by miomiocat — APPROVED** [review]

**Review by caneGuy — APPROVED** [review]

**Pull request merged**